metadata.yaml

```diff
@@ -3,29 +3,33 @@
 name: mysql-router
 display-name: MySQL Router
 maintainers:
-  - Shayan Patel <shayan.patel@canonical.com>
+  - Carl Csaposs <carl.csaposs@canonical.com>
   - Paulo Machado <paulo.machado@canonical.com>
+  - Shayan Patel <shayan.patel@canonical.com>
 description: |
   Machine charmed operator for mysql-router.
 summary: |
   Machine charmed operator for mysql-router.
   Enables effective access to a MySQL cluster with Group Replication.
 subordinate: true
 provides:
-  shared-db:
-    interface: mysql-shared
-    scope: container
+# TODO: re-implement legacy relation
+#  shared-db:
+#    interface: mysql-shared
+#    scope: container
   database:
     interface: mysql_client
     scope: container
 requires:
   backend-database:
     interface: mysql_client
+    limit: 1
+  # Workaround: Subordinate charms are required to have at least one `requires` endpoint with
+  # `scope: container`
```

Comment:
mysql_client interface already has scope: container.
JFYI, @dragomirp has scheduled a talk with Juju team about juju-info. Carl, consider to join us.